### PR TITLE
Do not printout basename usage

### PR DESCRIPTION
**Pull request by nadongguri**
When users command "bacardi" only, basename usage shouldn't come out.

ISSUE=#204

### CODE REVIEW

File: bootstrap/bacardi.sh [hunk 1]
```diff
@@ -40,7 +40,10 @@ if [ ! -f .last_update ] || [ package.json -nt .last_update ]; then
   npm install
 fi
 
+if [ "$1" ]; then
```
Comment (romandev) [inline bootstrap/bacardi.sh]: IMHO, if there is no subcommand, can we return early?

Comment (nadongguri) [reply bootstrap/bacardi.sh]: Do you mean not going through default task in gulp?

**Comment by nadongguri** [comment]
@romandev, if no subcommand, (ex: ./bacardi) doesn't it go through the default task then print out all the commands? If I get the wrong point, please fix me.

**Comment by romandev** [comment]
@nadongguri 
I think you can write the code as follows (pseudo code):
```
if there is a subcommand:
    processSubcommand()

processGulp()
```

**Comment by nadongguri** [comment]
@romandev, I got your point. :)

**Review by yjaeseok — APPROVED** [review]
lgtm

**Pull request merged**